infra/modules/service/access-logs.tf

```diff
@@ -21,7 +21,7 @@ locals {
 
 resource "aws_s3_bucket" "access_logs" {
   bucket_prefix = "${var.service_name}-access-logs"
-  force_destroy = false
+  force_destroy = true
```

Comment:
no biggie since this is just on the platform-test PR that won't get merged but looks like you accidentally checked this in

Reply:
Reverted. Good catch